gitbook-docs/market/economics.md

```diff
@@ -1,9 +1,11 @@
 # Economics
 
-{% hint style="danger" %}
+{% hint style="info" %}
 ### Disclaimer
```

Comment:
Headings go from level 1 straight to level 3. That's not right. Please move the headings up a level.